### PR TITLE
WIP: Ubuntu rework

### PR DESCRIPTION
**Pull request by juliogonzalez**
- Manual procedure for SUSE Manager (`spacewalk-common-channels` not supported yet)
- "Automated" procedure for Uyuni
- Fix the `if`s, now SUSE Manager goes first, Uyuni second, and the `if`s only happen for the content that is really different.

Feel free to edit directly at the PR for minor stuff. Reserve the comments only for doubts or big stuff.

### CODE REVIEW

**Comment by Loquacity** [comment]
OK, my edit is done :+1:

**Comment by keichwa** [comment]
On 2019-09-30 06:41, Lana Brindley wrote:
> OK, my edit is done :+1:
well done!  I added something minor.

-- 
Karl Eichwalder

**Comment by juliogonzalez** [comment]
What's missing here?

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -103,135 +66,106 @@ The repositories and channels for synchronizing upstream content must be configu
...
+
+
+There are two ways to check if a channel has finished synchronizing:
+
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
 +
-. Use the [command]``spacewalk-common-channels` command to create the required channels and repositories.
-Ensure you use the appropriate version number in this command, either [systemitem]``ubuntu-1604`` or [systeitem]``ubuntu-1804``:
+This dialog displays a completion bar for each product when they are being synchronized.
+* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]:
```suggestion
* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log``.
```

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 2]
```diff
@@ -33,67 +38,25 @@ Some actions are not yet supported:
...
+== Prepare to Register
 
 Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
 
-// ifndef works for displaying content when building uyuni
-// ifdef does not show the content for uyuni
-// ifeval works for displaying content
 
-ifeval::[{uyuni-content} == true]
+//ifeval::[{suma-content} == true]
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: Not sure, but this probably needs to be uncommented.

**Comment by Loquacity** [comment]
@juliogonzalez  are we able to merge these soon?

File: modules/client-configuration/pages/clients-ubuntu.adoc (hunk 2 above)
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: You are right.

**Comment by juliogonzalez** [comment]
I'd say: try to generate the PDFs, give it a final review, and if you don't find anything strange, we are ready to merge.

Again: let's squash the commits into a single one.

**Comment by Loquacity** [comment]
[suse_manager_client_configuration_guide.pdf](https://github.com/SUSE/doc-susemanager/files/3715588/suse_manager_client_configuration_guide.pdf)

**Comment by keichwa** [comment]
> I'd say: try to generate the PDFs, give it a final review, and if you don't find anything strange, we are ready to merge.

Because of all the ifevals, 1 PDF is not enough...

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 3]
```diff
@@ -103,135 +66,132 @@ The repositories and channels for synchronizing upstream content must be configu
...
+* In the [guimenu]``Repository URL`` field, type [systemitem]``http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/``.
+* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository]
+
+
+
+When you have created the repositories, you can create the custom channels.
+
+￼Ensure the custom channels you create have `AMD64 Debian` architecture.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]:
```suggestion
Ensure the custom channels you create have `AMD64 Debian` architecture.
```

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 4]
```diff
@@ -103,135 +66,132 @@ The repositories and channels for synchronizing upstream content must be configu
...
+* In the [guimenu]``Repository URL`` field, type [systemitem]``http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/``.
+* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository]
+. Click btn:[Create Repository] and set these parameters for the ``main-updates`` repository:
+* In the [guimenu]``Repository Label`` field, type [systemitem]``ubuntu-bionic-main-updates``.
+* In the [guimenu]``Repository URL`` field, type [systemitem]``http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/``.
+* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository]
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]:
```suggestion
. Click btn:[Create Repository].
```

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 5]
```diff
@@ -103,135 +66,132 @@ The repositories and channels for synchronizing upstream content must be configu
...
+[IMPORTANT]
+====
+You need all the new channels fully syncronized before bootstraping any Ubuntu client.
+====
 endif::[]
 
 
 
-.Procedure: Preparing to Register {ubuntu} Clients with Spacewalk
+ifeval::[{uyuni-content} == true]
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: I think this ifeval is problematic.

**Comment by keichwa** [comment]
> @keichwa is the PR being rejected because of "too many ifevals", or because of the "suma-content" without a title?

Both :)  I tried to indicate the most problematic ifeval.

File: modules/client-configuration/pages/clients-ubuntu.adoc (hunk 5 above)
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: I think you mean:
```
== Repository and Channel Management
```
Is not really needed. Right?

Please try to be more specific.

As for the other minor issues, I accepted the change but -as discussed- please fix such things on your own.

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: Or even that title show go above.

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: Should be fixed now.

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: See the PDFs below.

**Comment by juliogonzalez** [comment]
[suse_manager_client_configuration_guide.pdf](https://github.com/SUSE/doc-susemanager/files/3723519/suse_manager_client_configuration_guide.pdf)
[uyuni_client_configuration_guide.pdf](https://github.com/SUSE/doc-susemanager/files/3723521/uyuni_client_configuration_guide.pdf)

**Comment by Loquacity** [comment]
> 
> If it is the first one, then now it's for the doc team to evaluate how to fix that. Splitting the file or whatever.
> 

I think we probably should split the file, but we need to get this merged first then make the change.

**Pull request merged**